backend/views/syson-standard-diagrams-view/src/main/java/org/eclipse/syson/standard/diagrams/view/nodes/ActionDefinitionParameterBorderNodeDescriptionProvider.java

```diff
@@ -23,19 +24,32 @@
 import org.eclipse.syson.util.IDescriptionNameGenerator;
 
 /**
- * Used to create the item usage border node description.
+ * Used to create border node description for ActionDefinition getBehavior_Parameter.
  *
  * @author Arthur Daussy
  */
-public class ItemUsageBorderNodeDescriptionProvider extends AbstractItemUsageBorderNodeDescriptionProvider {
+public class ActionDefinitionParameterBorderNodeDescriptionProvider extends AbstractItemUsageBorderNodeDescriptionProvider {
 
-    public ItemUsageBorderNodeDescriptionProvider(IColorProvider colorProvider, IDescriptionNameGenerator descriptionNameGenerator) {
-        super(colorProvider, descriptionNameGenerator);
+    public ActionDefinitionParameterBorderNodeDescriptionProvider(EReference eReference, IColorProvider colorProvider, IDescriptionNameGenerator descriptionNameGenerator) {
+        super(eReference, colorProvider, descriptionNameGenerator);
+    }
+
+    @Override
+    protected String getName() {
+        return this.descriptionNameGenerator.getBorderNodeName(SysmlPackage.eINSTANCE.getItemUsage());
```

Comment:
eReference should probably be added as second parameter of the getBorderNodeName

Reply:
Could you check that please?